ci/docker/fasttest/Dockerfile

```diff
@@ -45,6 +45,7 @@ RUN curl -s https://apt.kitware.com/keys/kitware-archive-latest.asc | gpg --dear
 RUN apt-get update \
     && apt-get install \
         clang-${LLVM_VERSION} \
+        clang-tidy-${LLVM_VERSION} \
```

Comment:
clang-tidy now in a separate package?

Reply:
AFAIR there were no clang-tidy-19 in apt repos when I added this.